components/emoji_picker/modules/emoji_tabset.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div class="d-emoji-picker__tabset">
+    <dt-tab-group
+      tab-list-class="d-emoji-picker__tabset-list"
+      :selected="selectedTab"
+    >
+      <template #tabs>
+        <dt-tab
+          v-for="tab in tabs"
+          :id="tab.id"
+          :key="tab.id"
+          :panel-id="tab.panelId"
+          :label="tab.label"
+          class="d-ml2 d-mr2"
+          @click.capture.stop="selectTabset(tab.id)"
+        >
+          <dt-icon
+            size="400"
+            :name="tab.icon"
+          />
+        </dt-tab>
```

Comment:
hmm wonder if we could use dt-stack here rather than margin for spacing? or maybe that breaks the tab component...

Reply:
I tried to use dt-stack on the emoji_selector...but I didn't achieve the pixel perfect of design... Im still dealing with it so, I would prefer to keep it as this for now here also, I added this recommendation on https://dialpad.atlassian.net/browse/DT-1162

Reply:
k no problem then.